### ISSUE TITLE
travis's python nightly build is failed

### ISSUE THREAD

**Issue by podhmo** [podhmo/handofcats#46]
https://travis-ci.org/podhmo/handofcats/builds/634024576?utm_medium=notification&utm_source=github_status

**Comment by podhmo** [comment]
this is travis's issue

```
    ast3/Python/ast.c:4534:24: error: too many arguments to function ‘_PyBytes_DecodeEscape’
         PyObject *result = _PyBytes_DecodeEscape(s, len, NULL, 0, NULL,
                            ^
    In file included from /opt/python/3.9-dev/include/python3.9/Python.h:96:0,
                     from ast3/Python/ast.c:6:
    /opt/python/3.9-dev/include/python3.9/bytesobject.h:79:24: note: declared here
     PyAPI_FUNC(PyObject *) _PyBytes_DecodeEscape(const char *, Py_ssize_t,
                            ^
    error: command 'gcc' failed with exit status 1
```

**Comment by podhmo** [comment]
stop temporary
